lib/client.go

```diff
@@ -311,6 +311,21 @@ func GetUpdateVariablePayload(organization, workspaceName, variableID string, tf
 `, variableID, tfVar.Key, tfVar.Value, tfVar.Hcl, tfVar.Sensitive, organization, workspaceName)
 }
 
+// OrganizationExists returns whether an organization with the given name exists
+func OrganizationExists(organization string) (bool, error) {
+	if organization == "" {
+		return false, fmt.Errorf("OrganizationExists: organization is required")
+	}
+	u := NewTfcUrl("/organizations/" + organization)
+
+	resp := callAPI(http.MethodGet, u.String(), "", nil)
+
+	defer resp.Body.Close()
+
+	// Status 200 indicates the organization exists
+	return resp.StatusCode == 200, nil
```

Comment:
Would be good to avoid magic numbers.

```suggestion
	return resp.StatusCode == http.StatusOK, nil
```